pkg/istio/client.go

```diff
@@ -19,6 +19,8 @@ const (
 	firstElementIdx     = 0
 	vsDeletionThreshold = 1
 	contractVersion     = "v1"
+	prefixFormat        = "/%s/%s/event"
+	vsHost              = "*" // TODO: Needs to be changed to a valid hostname scheme!!!
```

Comment:
I now updated it with a valid hostname suffix matching. Yes it could be fetched using the gateway instead of hardcoding, but I would wait for this [issue](https://github.com/orgs/kyma-project/projects/30/views/2) to be resolved. Since the mentioned issue implement a mechanism to fetch the corresponding Gateway.

I just put the hostname as a constant here, since before it was hardcoded in-line and thus could be easily forgotten to adjust in future.